docs/tut-making-jettycat.md

```diff
@@ -577,21 +584,21 @@ if (ct.touch.collideUi(this)) {
 
 Remember the name `UI_Paused`. We will need to create a room with this name a bit later.
 
-`ct.pixiApp.ticker.speed` is the multiplayer that affects how ct.delta is calculated. When it is set to 0, it will effectively pause the game as everyone's `ct.delta` will turn to 0. Our cat and timers are dependant on `ct.delta`.
+`ct.pixiApp.ticker.speed` is the multiplier that affects how ct.delta is calculated. When it is set to 0, it will effectively pause the game as everyone's `ct.delta` will turn to 0. Our cat and timers are dependent on `ct.delta`.
 
 Open the room `UI_InGame` and place the created template at the top right corner.
 
-After that, create two new templates similar to those created for `MainMenu`. Use textures `Button_Play` and `Pause`. The button should be called `Button_Continue`, though.
+After that, create two new templates similar to those created for `MainMenu`. Use textures `Button_Play` and `Jetty_Cat`. The button should be called `Button_Continue`, though.
```

Comment:
Mmmmm nope, there is a texture `Pause` for the pause screen, different from the title screen